src/main/java/au/com/codeka/carrot/Bindings.java

```diff
@@ -0,0 +1,13 @@
+package au.com.codeka.carrot;
+
+/**
+ * The Bindings of a template when rendered.
+ *
+ * @author marten
+ */
+public interface Bindings
+{
+    Object resolved(String key);
```

Comment:
I think "resolve" (a verb) would be a better name.

Also, I would annotate this with @Nullable and document that null will be returned if the key doesn't exist.